Gifski/ConversionViewController.swift

```diff
@@ -116,12 +118,8 @@ final class ConversionViewController: NSViewController {
 			progress?.cancel()
 		}
 
-		stopConversion { [weak self] in
-			// It's safe to force-unwrap as there's no scenario where it will be nil.
-			let viewController = AppDelegate.shared.previousEditViewController!
-			viewController.isConverting = false
-
-			self?.push(viewController: viewController)
+		stopConversion { [self] in
```

Comment:
Same as https://github.com/sindresorhus/Gifski/pull/241#discussion_r627336411

Reply:
This one is a little different. The strong reference is required here so that the closure keeps ConversionViewController from being released during the transition.
There might be a better way that I haven't found yet to ensure this without requiring user code to keep a ref

Reply:
Must be the change in ownership model, before all view controllers ownership traced back to the edit view controller held by App